requirements/edx-sandbox/py35.in

```diff
@@ -7,6 +7,8 @@
 #   * confirm that it has no system requirements beyond what we already install
 #   * run "make upgrade" to update the detailed requirements files
 
+-c ../constraints.txt
```

Comment:
I don't know if we intentionally wanted separate constraints for the sandbox, and if kiwisolver constraint should be added here directly???

Reply:
py35.in does bring in shared.txt, which does bring in the constraints from constraint.txt file. So I suspect this omission was a mistake